common/web/lm-worker/build.sh

```diff
@@ -42,30 +42,19 @@ builder_parse "$@"
 # TODO: build if out-of-date if test is specified
 # TODO: configure if npm has not been run, and build is specified
 
-if builder_start_action configure; then
-  verify_npm_setup
-  builder_finish_action success configure
-fi
+builder_run_action configure verify_npm_setup
+builder_run_action clean rm -rf build/
 
-if builder_start_action clean; then
-  npm run clean
-  rm -rf ./build
-
-  builder_finish_action success clean
-fi
-
-if builder_start_action build; then
+do_build ( ) {
   # Build worker with tsc first
   tsc -b $builder_verbose || builder_die "Could not build worker."
+  node build-bundler.js
 
   EXT_FLAGS=
```

Comment:
```suggestion
function do_build() {
  # Build worker with tsc first
  tsc -b $builder_verbose
  node build-bundler.js

  local EXT_FLAGS=
```

`|| builder_die` hides exit codes and shouldn't be required